drivers/target/iscsi/iscsi_target_login.c

```diff
@@ -1177,7 +1177,7 @@ void iscsi_target_login_sess_out(struct iscsi_conn *conn,
 	if (!new_sess)
 		goto old_sess_out;
 
-	pr_err("iSCSI Login negotiation failed.\n");
+	pr_debug("iSCSI Login negotiation failed.\n");
 	iscsit_collect_login_stats(conn, ISCSI_STATUS_CLS_INITIATOR_ERR,
```

Comment:
This is interesting, where are these stats collected? Are these exposed through `/proc` somewhere?

Reply:
Good question, I haven't looked into it, but now I'm curious too.